zhaquirks/thirdreality/night_light.py

```diff
@@ -36,6 +37,12 @@ class ThirdRealitySpecificCluster(CustomCluster):
     """Manufacturer specific cluster to relay motion event to IAS Zone cluster."""
 
     cluster_id = THIRD_REALITY_CLUSTER_ID
+    # Add read-write interfaces for attributes in private cluster
+    attributes = {
```

Comment:
Since you're adding new attributes, please use the new syntax for defining them: https://github.com/zigpy/zha-device-handlers/blob/915ff9b41e7393e1f2eaa449bd6a09267fa84277/zhaquirks/legrand/cable_outlet.py#L51-L68